backend/src/main/java/gov/cdc/usds/simplereport/service/ApiUserService.java

```diff
@@ -16,28 +20,37 @@
 @Service
 @Transactional
 public class ApiUserService {
+    @Autowired
```

Comment:
We should pass this in through the constructor as well—mixing autowiring mechanisms is a recipe for confusion.